### PR TITLE
Add Placeholder to Twitter's Query

### PR DESCRIPTION
**Pull request by nikicc**
Added an explanation to Query Text Edit on how lines are joined.

### CODE REVIEW

**Comment by nikicc** [comment]
@mstrazar I added a placeholder that explains how lines are joined. Do you think this makes it clearer?
@s-alexey can you also check since this targets Twitter?

File: orangecontrib/text/widgets/owtwitter.py [hunk 1]
```diff
@@ -296,6 +298,7 @@ def update_api(self, key):
 
     @QtCore.pyqtSlot()
     @require('api', MISSED_KEY, MISSED_KEY_MESSAGE)
+    @require('word_list', MISSED_QUERY, MISSED_QUERY_MESSAGE)
```
Comment (s-alexey) [inline orangecontrib/text/widgets/owtwitter.py]: Because of `since` and `until` are always present `word_list` isn't required.

Comment (nikicc) [inline orangecontrib/text/widgets/owtwitter.py]: I cannot execute a query without any input in the `Query word list`. I am getting the error with code 403 and hence I added the check. 

Would this supposed to work, i.e. does tweepy even allow for request without any query and only time restrictions? Does this work for you?

Comment (s-alexey) [inline orangecontrib/text/widgets/owtwitter.py]: It's supposed to work, but I have got 403 too. But it works with empty authors ('from: ').
So, I will make a PR that fixes that.

Comment (nikicc) [inline orangecontrib/text/widgets/owtwitter.py]: Great! Then I'll remove it from this PR.

**Comment by ajdapretnar** [comment]
@nikicc I've tested this some more and I'm not receiving any tweets when I query 'big AND data' or 'data AND analytics'. Which should get me some results. Can anybody else check this as well? Whether logical AND works for queries? Tnx!

**Comment by nikicc** [comment]
@ajdapretnar I went and checked [the API](https://dev.twitter.com/rest/public/search) again and there is no AND operator. My bad 🙈.  I deleted the first sentence.

Anyhow this PR does not change the way searching works. It only provides the placeholder text.

**Comment by ajdapretnar** [comment]
Yes, I just noticed it when checking this PR. :grinning: It is indeed unrelated. As far as I'm concerned, we can merge this PR.

**Comment by nikicc** [comment]
I agree. We can still modify the message later if needed.

**Pull request merged**